From 752d360040428d9e014710912dbcebd48913f7a7 Mon Sep 17 00:00:00 2001
From: Carl Suster <carl@contraflo.ws>
Date: Wed, 4 Jan 2017 16:34:49 +1100
Subject: disable sphinx rst.linker extension

This is not yet packaged in Debian and is harmless to ignore for now.
---
 docs/conf.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index b14e42a..ff883b2 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -38,7 +38,6 @@ extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.todo',
     'sphinx.ext.intersphinx',
-    'rst.linker',
 ]
 
 intersphinx_mapping = {
